### PR TITLE
[service] Cleanup run bash script and add setup script for initialization

### PR DESCRIPTION
**Pull request by jonathanmetzman**

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
TBR

File: service/run.bash [hunk 1]
```diff
@@ -30,8 +31,7 @@ cd $repo_path
 make install-dependencies
 source .venv/bin/activate
 export PYTHONPATH=$repo_path
-cd $expriment_working_dir
 
+cd $expriment_working_dir
```
Comment (inferno-chromium) [inline service/run.bash:35]: typo expriment_working_dir missing 'e'

File: service/setup.bash [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  ./configure --enable-loadable-sqlite-extensions --enable-optimizations && \
+  sudo make -j install && \
+  sudo rm -r /tmp/Python-$PYTHON_VERSION.tar.xz /tmp/Python-$PYTHON_VERSION
+
+# Download and run the cloud_sql_proxy
+export cloud_sql_proxy_path=/tmp/cloud_sql_proxy
+wget https://dl.google.com/cloudsql/cloud_sql_proxy.linux.amd64 -O \
+  $cloud_sql_proxy_path
+chmod +x $cloud_sql_proxy_path
+# This is a hardcoded value that only works for the official fuzzbench service.
```
Comment (inferno-chromium) [inline service/setup.bash:50]: newline before this line.

File: service/setup.bash [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+cd /tmp/ && \
+  curl -O https://www.python.org/ftp/python/$PYTHON_VERSION/Python-$PYTHON_VERSION.tar.xz && \
+  tar -xvf Python-$PYTHON_VERSION.tar.xz && \
+  cd Python-$PYTHON_VERSION && \
+  ./configure --enable-loadable-sqlite-extensions --enable-optimizations && \
+  sudo make -j install && \
+  sudo rm -r /tmp/Python-$PYTHON_VERSION.tar.xz /tmp/Python-$PYTHON_VERSION
+
+# Download and run the cloud_sql_proxy
```
Comment (inferno-chromium) [inline service/setup.bash:45]: nit: end comment with period.

File: service/setup.bash [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Use this script once to setup a machine for running the fuzzbench service.
+
+# Install a supported python version.
+export PYTHON_VERSION=3.7.6
```
Comment (inferno-chromium) [inline service/setup.bash:19]: 3.7.8 is latest stable and should work.

Comment (jonathanmetzman) [reply service/setup.bash:19]: I'm using the same thing we use in the images. We should update it there first.

**Pull request merged**